client/src/utils.js

```diff
@@ -0,0 +1,44 @@
+export const formatDateforAPI = (currDate) => {
+  // getMonth returns 0-11
+  const currMonthStr = String(currDate.getMonth() + 1).padStart(2, "0");
+  // getDate returns 1-31
+  const currDayStr = String(currDate.getDate()).padStart(2, "0");
+  return currDate.getFullYear().toString() + currMonthStr + currDayStr;
+};
+
+export const buildDailyTaskRoute = (currDate) => {
+  return "/api/tasks/day/" + formatDateforAPI(currDate);
+};
+
+export const buildDateRangeRoute = (startDate, endDate) => {
+  return `/api/tasks?start=${formatDateforAPI(
+    startDate
+  )}&end=${formatDateforAPI(endDate)}`;
+};
+
+// TODO :
+export const convertTaskData = (fetchedTaskData) => {
```

Comment:
I would also start to write some boiler plate code for this (Even if you have to mock some data)

Reply:
Noted